docs/templates/product.html

```diff
@@ -0,0 +1,502 @@
+							FAQ content
+						</div>
+					</li>
+					<li class="py-3 border-bottom">
+						<button id="product-collapse-3__toggle" class="btn-unstyled w-100 d-flex justify-content-between font-weight-bold product-collapse__toggle collapsed sni-after sni__plus" type="button" data-toggle="collapse" data-target="#product-collapse-3" aria-expanded="true" aria-controls="product-collapse-3">Shipping &amp; Returns</button>
+						<div id="product-collapse-3" class="product-collapse collapse" aria-labelledby="product-collapse-3__toggle" data-parent="#product-collapse">
+							Shipping &amp; Returns content
+						</div>
+					</li>
+				</ul>
+
+				<ul class="list-unstyled mb-0 mt-4 mt-lg-5">
+					<li class="mb-2">
+						<form class="upsell product-upsell d-flex align-items-center p-0">
+							<figure class="d-flex flex-grow-1 mb-0">
+								<picture class="flex-shrink-0">
+									<img src="https://via.placeholder.com/180x180.jpg/EFADBA" class="w-100" />
+								</picture>
+								<figcaption class="d-flex flex-column mx-0 py-g pl-g px-lg-2">
+									<span class="upsell__title font-weight-bold">That’s a wrap bundle</span>
+									<span class="upsell__price text-primary flex-grow-1">
+										<span class="compare text-body">£XX.XX</span>
+										£XX.XX
+										<span class="save">(Save 20%)</span>
+									</span>
+								</figcaption>
+							</figure>
+							<a class="d-flex flex-column flex-shrink-0 align-items-center" role="button">
+								<i class="sni-text" aria-hidden="true">plus-circle</i>
+								<span class="product-upsell__atc-text text-primary font-weight-bold">Remove</span>
+							</a>
+						</form>
+					</li>
+					<li>
+						<form class="upsell product-upsell d-flex align-items-center p-0">
+							<figure class="d-flex flex-grow-1 mb-0">
+								<picture class="flex-shrink-0">
+									<img src="https://via.placeholder.com/180x180.jpg/EFADBA" class="w-100" />
+								</picture>
+								<figcaption class="d-flex flex-column mx-0 py-g pl-g px-lg-2">
+									<span class="upsell__title font-weight-bold">That’s a wrap bundle</span>
+									<span class="upsell__price text-primary flex-grow-1">
+										<span class="compare text-body">£XX.XX</span>
+										£XX.XX
+										<span class="save">(Save 20%)</span>
+									</span>
+								</figcaption>
+							</figure>
+							<a class="d-flex flex-column flex-shrink-0 align-items-center" role="button">
+								<i class="sni-text" aria-hidden="true">plus-circle</i>
+								<span class="product-upsell__atc-text text-primary font-weight-bold d-none d-lg-block">Add to Cart</span>
+							</a>
+						</form>
+					</li>
+				</ul>
+
+			</div>
+		</div>
+	</div>
+
+	<div class="container-fluid">
+		<div class="row">
+			<div class="col-12 col-lg-6 px-g px-lg-0 d-lg-flex align-items-center" style="background-color: #F9EECA7f;">
+				<div class="pdp-banner__content ml-lg-auto px-lg-g pt-4 pb-2 py-lg-0">
```

Comment:
we don't have a pdp-banner so pdp-banner__content can be just pdp-banner

Reply:
well, my intention is that those 2 content is a part of 1 banner, so i added `__content`. That class is actually to make half a container. Should i just rename it to `.container__half`?